### ISSUE TITLE
Interface sometimes disappears

### ISSUE THREAD

**Issue by dmeybohm** [dmeybohm/wisdom-chess#9]
I see this more consistently on the Web Assembly version, and usually after opening the menu for the first time. I've also seen it happen in the Android emulator.

It may have to do with the menu animation.

**Comment by dmeybohm** [comment]
This issue is fixed by https://github.com/dmeybohm/wisdom-chess/pull/19

**Comment by dmeybohm** [comment]
I saw this again in both Firefox and Chrome. The error in Chromeconsole is:

```
108WebGL: INVALID_OPERATION: drawElements: no buffer is bound to enabled attribute
``

In Firefox the console errors are:
```
WebGL warning: drawElementsInstanced: Vertex attrib array 2 is enabled but has no buffer bound. 29
After reporting 32, no further warnings will be reported for this WebGL context. 
```

**Comment by dmeybohm** [comment]
Fixed in #32
